config/clickhouse/dist_config.xml

```diff
@@ -0,0 +1,27 @@
+<yandex>
+    <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+
+    <remote_servers>
+        <cluster_one_sh>
+            <shard>
+                <replica>
+                    <host>localhost</host>
+                    <port>9000</port>
+                </replica>
+            </shard>
+        </cluster_one_sh>
+    </remote_servers>
+
+    <zookeeper>
+        <node>
+            <host>sentry_zookeeper</host>
+            <port>2181</port>
+        </node>
+    </zookeeper>
```

Comment:
I don't think the `sentry_zookeeper` name is right. The correct value is already be defined here https://github.com/getsentry/snuba/blob/master/config/clickhouse/zookeeper.xml#L4 so you should be able to remove this